src/Object/Upce.php

```diff
@@ -1,103 +1,109 @@
 <?php
 
-/**
- * @see       https://github.com/laminas/laminas-barcode for the canonical source repository
- * @copyright https://github.com/laminas/laminas-barcode/blob/master/COPYRIGHT.md
- * @license   https://github.com/laminas/laminas-barcode/blob/master/LICENSE.md New BSD License
- */
+declare(strict_types=1);
 
 namespace Laminas\Barcode\Object;
 
 use Laminas\Validator\Barcode as BarcodeValidator;
 
+use function implode;
+use function str_split;
+
 /**
  * Class for generate UpcA barcode
  */
 class Upce extends Ean13
 {
+    /** @var array */
```

Comment:
We should document this as `array<int, non-empty-list<non-empty-list<string>>>` or such